### PR TITLE
WIP: Fix maintenance

### PR DESCRIPTION
**Pull request by LimKianAn**
`maintenanceWindows` should be `string`. Several places must be moified.
```
apiVersion: acid.zalan.do/v1
kind: postgresql
metadata:
  labels:
    postgres.database.fits.cloud/uuid: a6dfb053-419e-456a-b5a8-b3779b19db23
  name: projectida-a6dfb053419e456ab5a8
  namespace: projectid-a
spec:
  maintenanceWindows:
  - Sun:22:00-23:00
  numberOfInstances: 2
  postgresql:
    version: "12"
  teamId: projectida
  volume:
    size: 1Gi
```

### CODE REVIEW

**Comment by majst01** [comment]
OK great, remove the  commented parts, then we should make this format accessible from the frontend(cloudctl) as well

**Comment by LimKianAn** [comment]
No more hacky resource. Only two resources: our `Postgres` and *zalando* `Postgresql`

File: api/v1/postgres_types.go [hunk 1]
```diff
@@ -361,95 +353,55 @@ func (p *Postgres) ToPeripheralResourceNamespace() string {
...
-	// todo: Check if the validation should be performed here.
 	z.Spec.PostgresqlParam.PgVersion = p.Spec.Version
-
-	z.Spec.ResourceRequests.CPU = p.Spec.Size.CPU
-	z.Spec.ResourceRequests.Memory = p.Spec.Size.Memory
-	z.Spec.ResourceLimits.CPU = p.Spec.Size.CPU
-	z.Spec.ResourceLimits.Memory = p.Spec.Size.Memory
-
-	// todo: Check if the validation should be performed here.
+	z.Spec.Resources.ResourceRequests.CPU = p.Spec.Size.CPU
```
Comment (majst01) [inline api/v1/postgres_types.go:372]: Size might be nil here

**Comment by majst01** [comment]
Like the removal of the extra zalando type, but i would prefer to have a pr which just fixes maintenance, and another one removing the zalando type.
WDYT ?

**Comment by LimKianAn** [comment]
Becuase of #144 and #147 we close this PR.

**Pull request closed without merging**